aries-site/src/layouts/main/NavLayout.js

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+
+import PropTypes from 'prop-types';
+import { Box, Main, ResponsiveContext } from 'grommet';
+
+import { SideBar, SideBarItemList } from '../navigation';
+
+const filterChildren = (children, type) => {
+  const filteredChildren = React.Children.map(children, child => {
+    return child.props[type] ? child : null;
+  });
+
+  if (filteredChildren.length > 1) {
+    console.warn(
+      `Expected a single ${type}, received ${filteredChildren.length}.`,
+      `Only first ${type} element will be rendered.`,
+    );
+  }
+
+  return filteredChildren;
+};
+
+export const NavLayout = ({ mainContentChilds }) => {
+  // Only expect a single child of the following types
+  const mainContent = filterChildren(mainContentChilds, 'MainContent');
+
+  // TODO selectedNav should be retrived from aries-core
+  // as the selected element of the NavBar
+  const selectedNav = 'start';
+  return (
+    <ResponsiveContext.Consumer>
+      {size => (
+        <>
+          <Box
+            direction="row"
+            // pad uses Nav pad from aries-core
+            pad={{
+              horizontal: size !== 'small' ? 'xlarge' : 'large',
+              vertical: 'large',
+            }}
+          >
+            {size !== 'small' && (
+              <Box fill="vertical">
+                <SideBar
+                  items={SideBarItemList[selectedNav]}
+                  prefix={selectedNav}
+                />
+              </Box>
+            )}
+            <Main flex>{mainContent[0]}</Main>
```

Comment:
I think the naming of 'NavLayout' made me think it was the layout of just the nav bar, not the pages with navigation so I got confused why 'Main' would be in this file. It makes sense now, but I'm wondering if there is a clearer name or a comment we could add to give context.

Reply:
In general, I think that the name Nav is confusing because the Header doesn't even have Nav on the new designs.
But I'm opened to other name suggestions... I'll take it on slack.